### DISCUSSION TITLE
Support Tagfiles

### DISCUSSION THREAD

**Discussion by Nerixyz** [JakubAndrysek/MkDoxy#106] [Ideas]
Doxygen has the option to specify [tagfiles which link to external documentation](https://www.doxygen.nl/manual/external.html). One example is Qt which hosts its tagfiles https://doc.qt.io/qt-6/qtcore.tags (and qtgui, qtwidgets, and so on). Currently, the generator fails to resolve these references.

**Comment by JakubAndrysek** [comment]
Thank you for sharing this information. MkDoxy currently does not support all tags. Adding more parsers and processing would be excellent, but I currently don't have time for that. 

Feel free to implement it.
